rpc/examples/echo/node-client/src/index.ts

```diff
@@ -0,0 +1,108 @@
+import type { PartialMessage } from "@bufbuild/protobuf";
+import { ConnectError, createPromiseClient, PromiseClient } from "@connectrpc/connect";
+import { createWritableIterable } from "@connectrpc/connect/protocol";
+import { dialDirect, dialWebRTC } from "@viamrobotics/rpc";
+import { EchoService } from "./gen/proto/rpc/examples/echo/v1/echo_connect.js";
+import { EchoBiDiRequest, EchoMultipleRequest, EchoRequest } from "./gen/proto/rpc/examples/echo/v1/echo_pb.js";
+
+import { createGrpcTransport } from '@connectrpc/connect-node';
```

Comment:
@njooma this has webrtc "just work" but as far as the TS SDK, I feel like this is kind of ugly and warrants its own node sdk package, unless viam is okay with everyone needing to do this in node. there could be a helper that we may be able to get away with using dynamic imports (that you await)

Reply:
no opinion on whether this should be it's own SDK package or not, but it's nice that node is supported in _some_ way with this change